lib/ip-locator.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const Ipinfo = require('ipinfo')
+const RequestIp = require('request-ip')
+
+class IpLocator {
+  constructor (options) {
+    const { enabledByDefault = true } = options
+    this.enabledByDefault = enabledByDefault
+  }
+
+  /**
+   * @function setting
+   * @description Return the setting for the hapi-geo-locate by taking the request as a parameter and return the setting
```

Comment:
This comment is a "cookbook comment". It phrases exactly the method definition. This type of comment is helpful for a task description, but it can be skipped here